src/lib.rs

```diff
@@ -12,8 +12,6 @@ pub use database::Options;
 pub use database::new;
 pub use database::*;
 
-pub mod tests;
```

Comment:
if this module were a Unit test then you'd use #[cfg(test)] to conditionally enable this, only at test time